src/util/debug.h

```diff
@@ -16,29 +16,29 @@
 #endif
 
 // Initial setup of debug; only to be used once
-#define DEBUG_BEGIN DEBUG_SERIAL_PORT.begin(115200)
+#define DEBUG_BEGIN DEBUG_SERIAL_PORT.begin(230400)
```

Comment:
Any reason in particular for selecting this baud rate in particular?

Reply:
Trying to avoid `DEBUG_PRINT`s from slowing down the system as much as possible (sending say a 64 byte message at 115200 takes about ~5.6ms). Internet suggests anything above 230400 doesnt work on Linux systems for the Arduino IDE (and 250000 on Windows), which was true for me (250000 worked, 500000 did not). So, just bumped it up to the closest thing that works. Since we are trying to run our control loop at 100Hz, wanted to do the most we could to avoid problems with prints (thus the phase adjustments to `DEBUG_PRINT_EVERY`)

Reply:
I figured that this was the maximum reasonable baud rate for the project. No complaints here.